openshift/templates/jobs/mongo-migrations/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM alpine:latest
+USER root
+
+# install glibc for mongo/oc binaries
+ENV LANG=C.UTF-8
+RUN ALPINE_GLIBC_BASE_URL="https://github.com/sgerrand/alpine-pkg-glibc/releases/download" && \
+    ALPINE_GLIBC_PACKAGE_VERSION="2.31-r0" && \
+    ALPINE_GLIBC_BASE_PACKAGE_FILENAME="glibc-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    ALPINE_GLIBC_BIN_PACKAGE_FILENAME="glibc-bin-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    ALPINE_GLIBC_I18N_PACKAGE_FILENAME="glibc-i18n-$ALPINE_GLIBC_PACKAGE_VERSION.apk" && \
+    apk add --no-cache --virtual=.build-dependencies wget ca-certificates && \
+    echo \
+        "-----BEGIN PUBLIC KEY-----\
+        MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApZ2u1KJKUu/fW4A25y9m\
```

Comment:
What is this public key?

Reply:
https://github.com/sgerrand/alpine-pkg-glibc/

Public signing key for the compiler. Could be replaced with a call to the url instead of hard-coded in, but hard-coding prevents issues if the key is not available online for whatever reason.

You can find the key here: https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub

Reply:
Ahh, okay. If it ever changes we will need to update, but that makes sense.